tutorial/creating_packages/add_requires/conanfile.py

```diff
@@ -24,6 +24,7 @@ class helloRecipe(ConanFile):
 
     def validate(self):
         check_min_cppstd(self, "11")
+        check_max_cppstd(self, "14")
```

Comment:
The compiler is using 17 by default for some configurations so let's raise this to 23 (we also have to do a PR for the docs and update it or it will fail for users following the tutorial)
```suggestion
        check_max_cppstd(self, "23")
```